### PR TITLE
Create recipe for pdsspect

### PR DESCRIPTION
**Pull request by pbvarga1**
Add pddspect to conda forge

### CODE REVIEW

**Comment by pbvarga1** [comment]
I'm assuming that this fails since ginga is not a conda forge package. If so, go ahead and close PR. Otherwise, can I get help as to why it is breaking?

**Comment by synapticarbors** [comment]
@pbvarga1 `ginga` is indeed missing from conda-forge. You can click the `Details` link next to any of the CI services and look at the log file to see why the build is failing. 

There is no need to close this PR if you (or someone else) is willing to put together the recipe for `ginga`. Once that recipe is merged we could return to this PR.

**Comment by mwcraig** [comment]
I can move ginga over later today...will be starting a migration of ~40 recipes form the astropy channel to conda-forge.

**Comment by pbvarga1** [comment]
@mwcraig Thanks! Would that include ``ginga==2.6.0``? I can't get the latest version to work so I need 2.6.0

**Comment by mwcraig** [comment]
Sure, I can start with that one. Please do report the ginga bug at https://github.com/ejeschke/ginga if you haven't yet.

**Comment by mwcraig** [comment]
@pbvarga1 -- if you need ginga 1.6.0 today, then you can install it from the astropy channel.

**Comment by pbvarga1** [comment]
Definitely planning on reporting the bug. Just need some time to make sure I'm not doing anything wrong on my end and create simpler example that exposes the problem

**Comment by mwcraig** [comment]
closing/reopening to trigger CIs now that ginga is available.

File: recipes/pdsspect/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - ginga 2.6.0
```
Comment (synapticarbors) [inline recipes/pdsspect/meta.yaml]: It doesn't look like `ginga` is required to build the package. Please check and if this is the case remove it from the `build` requirements.

File: recipes/pdsspect/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "pdsspect" %}
+{% set version = "0.1.0" %}
```
Comment (synapticarbors) [inline recipes/pdsspect/meta.yaml]: Update to `0.1.1`?

File: recipes/pdsspect/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (synapticarbors) [inline recipes/pdsspect/meta.yaml]: It looks like there is a cli. Please add it here. See:

https://conda.io/docs/building/meta-yaml.html#python-entry-points

File: recipes/pdsspect/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  run:
+    - python
+    - planetaryimage
+    - matplotlib
+    - ginga 2.6.0
+    - qtpy
+
+test:
+  imports:
+    - pdsspect
```
Comment (synapticarbors) [inline recipes/pdsspect/meta.yaml]: Add a command test for the cli:

https://conda.io/docs/building/meta-yaml.html#test-commands

**Pull request closed without merging**